builder/proxmox/iso/config_test.go

```diff
@@ -103,8 +103,8 @@ func TestBasicExampleFromDocsIsValid(t *testing.T) {
 	if b.config.Disks[0].CacheMode != "none" {
 		t.Errorf("Expected disk cache mode to be 'none', got %s", b.config.Disks[0].CacheMode)
 	}
-	if b.config.Agent.True() != true {
-		t.Errorf("Expected Agent to be true, got %t", b.config.Agent.True())
+	if b.config.GuestAgent.Enabled.True() != true {
```

Comment:
I would suggest simplifying this one; 
```suggestion
	if !b.config.GuestAgent.Enabled.True() {
```

Reply:
Done, added to reroll